src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -0,0 +1,100 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import "@polymer/paper-checkbox/paper-checkbox.js";
+
+import processConfigEntities from "../common/process-config-entities";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../types.js";
+import { LovelaceCardEditor } from "../types.js";
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { Config, EntityConfig } from "../cards/hui-glance-card";
+
+import "../../../components/entity/state-badge.js";
+import "../../../components/entity/ha-entity-picker";
+import "../../../components/ha-card.js";
+import "../../../components/ha-icon.js";
+import { TemplateResult } from "lit-html";
```

Comment:
🙄

Reply:
REMEMBER TO GO BACK AND CHECK ZACK, Idiot